pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -95,10 +95,7 @@ func containerLabels(c *cadvisorapi.ContainerInfo) map[string]string {
 
 // New creates a cAdvisor and exports its API on the specified port if port > 0.
 func New(port uint, runtime string, rootPath string) (Interface, error) {
-	sysFs, err := sysfs.NewRealSysFs()
-	if err != nil {
-		return nil, err
-	}
+	sysFs := sysfs.NewRealSysFs()
```

Comment:
thats because in recent version there is change in NewRealSysFS function, check this out for more information - https://github.com/google/cadvisor/pull/1578